Added information on unregistering printing events

jamesmcroft opened the pull request:
As #1982 describes, the documentation for using the printing APIs in a UWP application shows how a developer can easily set up the registration of the events needed to performing a print but don't currently show an example of what needs unregistering.

This change takes a code snippet from the [UWP Print Sample](https://github.com/microsoft/Windows-universal-samples/blob/master/Samples/Printing/cs/PrintHelper.cs) on unregistering the events, with a reword of the details to match the register section.

jamesmcroft committed:
Added information on unregistering printing events

PRMerger13 commented:
@jamesmcroft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Karl-Bridge-Microsoft committed:
Update print-from-your-app.md

some revisions. Replace fwlinks with specific links, various rewrites/rewording.

PRMerger17 commented:
@Karl-Bridge-Microsoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Karl-Bridge-Microsoft commented:
#sign-off

Karl-Bridge-Microsoft commented:
@jamesmcroft - thanks for the feedback and suggestions!
